tensor2tensor/data_generators/gym_env.py

```diff
@@ -24,6 +24,8 @@
 import random
 
 import numpy as np
+from PIL import Image
```

Comment:
Let's try to do this in TF in a similar way to image encoding. I actually don't have a preference here, but there are many resizing methods and many implementation differences. We know our current tests run well with tf.resize with method=AREA so I'd rather keep it the same -- the less we change, the less potential causes if it doesn't work after refactor.